modules/govuk/manifests/apps/frontend.pp

```diff
@@ -28,23 +24,12 @@
 #
 class govuk::apps::frontend(
   $port = '3005',
-  $enable_homepage_nocache_location = true,
   $vhost_protected = false,
   $publishing_api_bearer_token = undef,
   $nagios_memory_warning = undef,
   $nagios_memory_critical = undef,
 ) {
-  validate_bool($enable_homepage_nocache_location)
-
-  if ($enable_homepage_nocache_location) {
-    $nginx_extra_config = '
-  location ^~ /frontend/homepage/no-cache/ {
-    expires epoch;
-  }
-'
-  } else {
-    $nginx_extra_config = ''
-  }
+  $nginx_extra_config = ''
```

Comment:
You can remove this line and the parameter from the `govuk::app` declaration below, as the default is `''`